README.md

```diff
@@ -0,0 +1,70 @@
+# express-mockups-middleware
+
+## What
+
+This is an express `middleware`.
+
+This middleware purpose is to serve mockups on proxified HTTP requests.
+
+This is convenient during development to separate concerns.
+
+## Install
+
+`npm i --save-dev express-mockups-middleware`
+
+## use
+
+You should pass as first parameter of the middleware an array of apis. api objects must respect the following structure.
```

Comment:
Pour moi dans ce contexte, have to et should c'est ok. Par contre je tournerai la phrase :
`You should pass an array of apis as first parameter of the middleware`